### ISSUE TITLE
Release v2.0 to Supermarket?

### ISSUE THREAD

**Issue by mattstratton** [lynx44/windows_firewall#12]

**Comment by lynx44** [comment]
Yeah, I looked into this briefly the other day. Problem is that I don't have a pem file to authenticate with the Supermarket, as I created this account before that was a thing (you used to be able to upload a zip file through the web interface).

There doesn't seem to be a lot of information on how the supermarket works in general, so I'm either going to need to create another account and add it as a contributor or transfer ownership, or I may need to get in touch with Chef to see what I can do to get a pem file.

**Comment by mattstratton** [comment]
You should be able to reset your pem (supermarket uses the same OCID as hosted chef) so if you log into manage.chef.io, you can reset your key, which gives you a new username.pem, etc. 

I'll do a little digging. As it happens, I work for Chef, so I can help :)

**Comment by mattstratton** [comment]
You can also add me as a contributor to the cookbook via Supermarket if you'd like; my chef username is mattstratton.

**Comment by lynx44** [comment]
Do I need to install ChefDK? We're still on chef 11 internally, we haven't had the time or need to update. 

When I run this command, it tells me I'm not authorized:

knife cookbook site share windows_firewall Networking

I've updated my knife.rb file with my chef username and pem, and I've logged into manage.chef.io and supermarket.chef.io through my browser.

Things have changed quite a bit since I've dealt with this stuff, and I have to admit it's a bit frustrating to find good information about how this is all supposed to work. I haven't had a lot of success googling chef supermarket concepts. We also host our own chef instance internally, so I wasn't familiar with manage.chef.io.

Anyway, I'm just going to add you as a contributor since I'm sure you can get this up quickly.

**Comment by mattstratton** [comment]
Yeah, one reason stove is recommended is that the `knife cookbook site share` stuff kind of assumes the monolithic chef-repo model. 

I'll just publish it. You can have the bragging rights of announcing it on the chef mailing list once it is done though :)

**Comment by mattstratton** [comment]
Published to the supermarket! Whoo!

I'll be submitting a PR that doesn't affect the published version, that just bumps the test-kitchen version since it's now been merged into master.

**Comment by lynx44** [comment]
Thanks, your latest PR has been merged. 

By the way, do you want to be added as a collaborator on this project?

**Comment by mattstratton** [comment]
Sure, if you don't mind. 

Sent from Outlook




On Fri, Mar 27, 2015 at 4:06 PM -0700, "Matt Clifton" <notifications@github.com> wrote:












Thanks, your latest PR has been merged. 



By the way, do you want to be added as a collaborator on this project?
